knowledge_storm/rm.py

```diff
@@ -13,6 +13,8 @@
 
 from .utils import WebPageHelper
 
+from azure.core.credentials import AzureKeyCredential
```

Comment:
Could you move this inside `AzureAISearch`? You can add the following code (check out `GoogleSearch` for example)
```python
try:
    from azure.core.credentials import AzureKeyCredential
    from azure.search.documents import SearchClient
except ImportError as err:
    raise ImportError("AzureAISearch requires `pip install azure-search-documents`.") from err
```

Reply:
included this change